ddev/src/ddev/cli/validate/licenses.py

```diff
@@ -467,7 +470,7 @@ def licenses(app: Application, sync: bool):
         for package_name, package_errors in package_license_errors.items():
             for error in package_errors:
                 error_message += error + '\n'
-            validation_tracker.error((package_name), message=error_message)
+            validation_tracker.error((package_name,), message=error_message)
```

Comment:
Is this change intended? I guess this make this expression a tuple

Reply:
Yes, this is what is causing the display issue. Without the comma python assumes the parenthesis is just a scoping marker and when we loop over them (since the argument needs to be a tuple) we get this error message

```
Licenses
└── u
    └── r
        └── l
            └── l
                └── i
                    └── b
                        └── 3
                            
                            no license information
```